monitor/Errors.ts

```diff
@@ -3,7 +3,22 @@ export interface CodeChainAlert {
   content: string;
 }
 
-export const codechainDeath: CodeChainAlert = {
-  title: "[CodeChain Death Confirmation]",
-  content: "CodeChain didn't renew the best block number for 1 hour."
-};
\ No newline at end of file
+export class CodeChainDeath implements CodeChainAlert {
+  public title: string;
+  public content: string;
+
+  constructor() {
+    this.title = "[CodeChain Death Confirmation]";
```

Comment:
Would you add the network id on the title?
I want to show the title looks like `[cc network Death Confirmation]` or `[wc network Death Confirmation]`

Reply:
prefix added